pkg/config/validation/validation_test.go

```diff
@@ -167,6 +167,20 @@ func TestValidatePort(t *testing.T) {
 	}
 }
 
+func TestValidateControlPlaneAuthPolicy(t *testing.T) {
+	policys := map[meshconfig.AuthenticationPolicy]bool{
```

Comment:
In go, table driven test is recommended 
https://github.com/golang/go/wiki/TableDrivenTests

Reply:
done